keras/backend/common.py

```diff
@@ -63,27 +161,87 @@ def set_image_dim_ordering(dim_ordering):
 
 
 def get_uid(prefix=''):
+    '''
+
+    # Argument
+        prefix: string
+
+    # Return
+        integer (scalar)
+
+    # Example
+    ```
+        >>> a = keras.backend.get_uid()
+        >>> type(a)
+        <type 'int'>
+    ```
+
+    '''
     _UID_PREFIXES[prefix] += 1
     return _UID_PREFIXES[prefix]
 
 
 def reset_uids():
+    '''
+    # Return
+        `None`
```

Comment:
Could you provide docstrings for `get_uid()`, `reset_uids()`, `set_legacy_weight_ordering()`, and `legacy_weight_ordering()`? I don't understand what they do.